test/12-logger.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('Logger', function () {
 		this._req.end = sinon.spy();
 
 		this._req.body = '';
-		this._req.pipe(zlib.createGunzip()).on('data', (chunk) => {
+		this._req.pipe(zlib.createGunzip()).on('data', (chunk: any) => {
```

Comment:
I think this should be a `Buffer`. In general I'd like to remove the vast majority of `any`s from the code.